### PR TITLE
Race+headshots (#19)

### PR DESCRIPTION
**Pull request by RyanM1235**
* headshots

* additionalheadshots +edits

### CODE REVIEW

File: src/pages/race.astro [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+</main>
+<Layout title="The Next Race">
+    <main>
+      <Image
+        src={bunny}
+        alt="Formula Nano Team Members on a sunny day standing in front of University of Waterloo Quantum Nano Center"
+        class="mt-8 h-48 w-full rounded-2xl object-cover sm:h-[30rem] object-top"
+      />
+      <h1 class="text-center py-8 text-5xl font-bold">Our Race to the Next Nanocar Competition!</h1>
```
Comment (bergermaestro) [inline src/pages/race.astro]: This should match the headings on other pages (larger text size and left aligned)

File: src/pages/race.astro [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  
+        <Image
+          src={carimage}
+          alt="Car and molecule combination picture"
+          class="mt-8 h-64 w-96 rounded-2xl object-cover mx-auto shadow-lg transform hover:scale-105 transition-all duration-300"
+        />
+      </section>
+
+       <!-- Competition Rules Section -->
+    <section class="mt-8 bg-gradient-to-r from-red-100 to-yellow-100 py-12 px-6 sm:px-12 rounded-2xl object-cover shadow-xl hover:scale-105 transition-all duration-300">
```
Comment (bergermaestro) [inline src/pages/race.astro]: What's the deal with the scaling on hover?

File: src/data/team_members.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   {
     "name": "Ryan Miller",
     "team": "Computation Lead",
-    "profile_url": "https://dummyjson.com/icon/alice/151"
+    "profile_url": "src/data/images/headshots/ryanmiller.png"
```
Comment (bergermaestro) [inline src/data/team_members.json]: All of these URLs don't load properly on the about page. It's trying to load `/about/src/data/images/headshots/[filename]`

File: tsconfig.json [hunk 4]
```diff
@@ -3,5 +3,7 @@
   "compilerOptions": {
     "jsx": "react-jsx",
     "jsxImportSource": "react"
-  }
+  },
+  "include": ["src/**/*.ts", "src/**/*.tsx", "src/**/*.astro"],
+  "exclude": ["node_modules", "src/pages/test.astro"]
```
Comment (bergermaestro) [inline tsconfig.json]: Why was this change needed, and why does it include `test.astro`, a file that doesn't exist?

**Review by bergermaestro — CHANGES_REQUESTED** [review]

File: src/pages/race.astro (hunk 2 above)
Comment (jasminetey) [reply src/pages/race.astro]: it makes the images "pop out"/larger when hovered

File: tsconfig.json (hunk 4 above)
Comment (jasminetey) [reply tsconfig.json]: oh this was part of a separate edit- will remove

**Comment by bergermaestro** [comment]
Same changes implemented in #21
